### PR TITLE
Xdr translate helper functions

### PR DESCRIPTION
**Pull request by kbartush**
Adding two utility functions that will help translate between formats for the common xdr objects in mazzaroth-xdr.

### CODE REVIEW

**Comment by kbartush** [comment]
I found a bug when I was looking over this pull request. I'm removing you guys form review while I fix it.

**Comment by kbartush** [comment]
> I found a bug when I was looking over this pull request. I'm removing you guys form review while I fix it.

Should be properly handling dictionary merging now.

File: src/client/utils.js [hunk 1]
```diff
@@ -114,6 +115,47 @@ export function RunExecutionPlan (plan, privKey, progress, client) {
...
+/**
+ * Translates an xdr string to a json string  XDR representation of the
+ * supplied data  type.
+ *
+ * @param input string xdr representation matching the arg format
+ * @param type string xdr object from mazzaroth-xdr i.e. 'Transaction'
+ * @param format data string format base64/hex defaults to base64
+ * @return string json string representation of XDR object
+*/
+export function XDRtoJSON (input, type, format) {
```
Comment (jeff1010322) [inline src/client/utils.js:147]: I don't understand how format is being used in this function.  What is the format provided supposed to do?

Comment (kbartush) [reply src/client/utils.js:147]: I meant to allow for the passing of hex when I originally wrote the function. We don't really use that much so I forgot to actually implement it. Should be good now.

**Pull request merged**